docs/user-tutorial/benchmarks/micro-benchmarks.md

```diff
@@ -95,20 +95,20 @@ or [AMD](https://github.com/ROCm-Developer-Tools/HIP/tree/master/samples/1_Utils
 
 #### Metrics
 
-| Name              | Unit             | Description                      |
-|-------------------|------------------|----------------------------------|
-| mem-bw/H2D_Mem_BW | bandwidth (GB/s) | Host to device copy bandwidth.   |
-| mem-bw/D2H_Mem_BW | bandwidth (GB/s) | Device to host copy bandwidth.   |
-| mem-bw/D2D_Mem_BW | bandwidth (GB/s) | Device to device copy bandwidth. |
+| Name       | Unit             | Description                      |
+| ---------- | ---------------- | -------------------------------- |
+| mem-bw/h2d | bandwidth (GB/s) | Host to device copy bandwidth.   |
+| mem-bw/d2h | bandwidth (GB/s) | Device to host copy bandwidth.   |
+| mem-bw/d2d | bandwidth (GB/s) | Device to device copy bandwidth. |
 
 ### `gpu-copy-bw`
 
 Measure the memory copy bandwidth performed by GPU SM/DMA engine, including device-to-host, host-to-device and device-to-device.
 
 #### Metrics
 
-| Name                                                                       | Unit             | Description                                                                                                                | 
-|----------------------------------------------------------------------------|------------------|----------------------------------------------------------------------------------------------------------------------------|
+| Name                                                                       | Unit             | Description                                                                                                                |
+| -------------------------------------------------------------------------- | ---------------- | -------------------------------------------------------------------------------------------------------------------------- |
 | cpu\_to\_gpu[0-9]+\_by\_gpu[0-9]+\_using\_(sm\|dma)\_under_numa[0-9]+      | bandwidth (GB/s) | The bandwidth reading from all NUMA nodes' host memory using DMA engine or GPU SM by all GPUs.                             |
```

Comment:
cpu\_to\_gpu[0-9]+\_by\_gpu[0-9]+\_using\_(sm\|dma)\_under_numa[0-9]+_bw

Reply:
@abuccts please help check it.

Reply:
`\` here is used to resolve formatting issue, you can check [rendered page](https://microsoft.github.io/superbenchmark/docs/user-tutorial/benchmarks/micro-benchmarks#gpu-copy-bw)